src/libraries/System.Net.Http/src/System/Net/Http/Headers/NameValueHeaderValue.cs

```diff
@@ -180,7 +180,6 @@ private void AddToStringBuilder(StringBuilder sb)
                 if (leadingSeparator || (destination.Length > 0))
                 {
                     destination.Append(separator);
-                    destination.Append(' ');
```

Comment:
This is going to affect much more than just Accept.  That's ok?

Also, I'm surprised no tests need to change; we don't have any tests validating the output here?

@davidsh, you had several comments on the original issue stating the current behavior was desirable; did we reverse our decision on that?

Reply:
@stephentoub , I tested and researched some http request scenarios and there was no problem. Can you identify any scenarios that this change may cause problems?
If you deem it necessary to maintain the current behavior, I create an argument to use the space between the headers or not, so there will be no impact on the current function.

Reply:
I'm still seeing this file in the PR:
 src/installer/pkg/packaging/snaps/dotnet-sdk/snap/snapcraft.yaml

Reply:
>This is going to affect much more than just Accept. That's ok?

I don't think this change is the correct one to make. It is overly broad and will affect a lot of headers.